lib/modules/datasource/go/releases-goproxy.spec.ts

```diff
@@ -195,46 +205,54 @@ describe('modules/datasource/go/releases-goproxy', () => {
       });
     });
 
-    it('handles pipe fallback', async () => {
-      process.env.GOPROXY = `https://example.com|${baseUrl}`;
-
-      httpMock
-        .scope('https://example.com/github.com/google/btree')
-        .get('/@v/list')
-        .replyWithError('unknown');
-
-      httpMock
-        .scope(`${baseUrl}/github.com/google/btree`)
-        .get('/@v/list')
-        .reply(
-          200,
-          codeBlock`
+    it.each`
```

Comment:
you can still have type support

https://github.com/renovatebot/renovate/blob/d65effaa77b53ef93312474e81fdbd0efeff0498/lib/modules/datasource/nuget/common.spec.ts#L4